buildenv/jenkins/JenkinsfileBase

```diff
@@ -656,11 +663,18 @@ def buildTest() {
 			// check for each lib. If lib does not exist, donwload it.
 			// If lib exists, SHA will be checked. Re-download if SHA does not match.
 			timeout(time: 20, unit: 'MINUTES') {
-				def customUrl = "https://ci.adoptium.net/job/test.getDependency/lastSuccessfulBuild/artifact/"
+				def defaultUrl = "https://ci.adoptium.net/job/test.getDependency/lastSuccessfulBuild/artifact/"
+				def systemUrl = "https://ci.adoptium.net/job/systemtest.getDependency/lastSuccessfulBuild/artifact/"
+				env.LIB_DIR = (env.BUILD_LIST == 'system') ? env.SYSTEM_LIB_DIR : env.LIB_DIR
+				def customUrl = (env.BUILD_LIST == 'system') ? systemUrl : defaultUrl
 				if (PLATFORM.contains("windows")) {
 					env.LIB_DIR = env.LIB_DIR.replaceAll("\\\\", "/")
 				}
-				sh "perl ./aqa-tests/TKG/scripts/getDependencies.pl -path ${env.LIB_DIR} -task default -customUrl ${customUrl}"
+				if (env.BUILD_LIST != 'system') {
+					sh "perl ./aqa-tests/TKG/scripts/getDependencies.pl -path ${env.LIB_DIR} -task default -customUrl ${customUrl}"
+				} else {
+					sh "perl ./aqa-tests/TKG/scripts/getDependencies.pl -path ${env.LIB_DIR} -task default"
+				}
```

Comment:
for system tests, we do not need customUrl?

Reply:
Passing the customUrl for system tests was duplicating the process of downloading the jar files.

Reply:
why having customUrl will duplicate the process? When running at openj9 Jenkins, we should try to use systemtest.getDependency from openj9 Jenkins, not Adoptium.

Reply:
@llxia I have used customUrl and replaced `test.getDependency` to `systemtest.getDependency` for system test libs in TKG.

Reply:
Grinder links:
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/40841/